### PR TITLE
[Backend] Contact Us Form [Post]

### PR DESCRIPTION
**Pull request by tejasvi541**
## Issue that this pull request solves

 Closes: #671

 ## Proposed changes
 * Added new route as "/contactus" which will add new join us requests data to DB.
 * Sendgrid email notifications integrated.
 * New MongoDB model for Contact Us request added.
 

 ## Types of changes
 _Put an `x` in the boxes that apply_
 
 * [x]  New feature (non-breaking change which adds functionality)

 ## Checklist
 _Put an `x` in the boxes that apply_
 
 * [x]  My code follows the style guidelines of this project
 * [x]  I have performed a self-review of my own code
 * [x]  My changes generate no new warnings
 * [x]  My changes does not break the current system and it passes all the current test cases.
 

## Other information

> Use your own sendGrid credentials for testing the api.
> This PR is under SWOC

### CODE REVIEW

File: backend/app/routes/contactUs/post.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const to = require('await-to-js').default;
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+const CONTACT = require('../../models/contactUs');
```
Comment (Kajol-Kumari) [inline backend/app/routes/contactUs/post.js:4]: please replace `CONTACT` with `contactUs`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@tejasvi541 minor change required, rest Looks good to go!

File: backend/app/routes/contactUs/post.js (hunk 1 above)
Comment (tejasvi541) [reply backend/app/routes/contactUs/post.js]: Requested changes done !

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**